packages/wdio-sync/src/executeHooksWithArgs.ts

```diff
@@ -9,10 +9,10 @@ const log = logger('@wdio/sync')
  * actual test process.
  *
  * @param  {Function|Function[]} hooks  list of hooks
- * @param  {Object[]} args  list of parameter for hook functions
+ * @param  {object|object[]} args  list of parameter for hook functions
  * @return {Promise}  promise that gets resolved once all hooks finished running
  */
-export default function executeHooksWithArgs (hooks = [], args) {
+export default function executeHooksWithArgs<T> (hooks: Function | Function[] = [], args: object | object[] = {}): Promise<(T | Error)[]> {
```

Comment:
can you please replace `object` with `Record<...>` or even define what kind of object is `args`

Reply:
The problem is the jsdoc mentioned `Object[]` before my changes, but I discovered that `Object` is accepted too (the code also handles this one). If I change it from object to something else, wouldn't that be a breaking change?

Reply:
Jsdoc in the end is just a comment

Reply:
Definitely true, but still if someone assumed it to be an object (uses typescript for example, and has their own methods that use it typed as `object` due to the jsdoc) they will probably be surprised after upgrading about the error due to the different type. I would be fine with changing it, if @christian-bromann agrees?